src/components/MegaButton/MegaButton.vue

```diff
@@ -14,6 +14,7 @@
     <label
       :for="id"
       :class="[
+        'text-gray-900',
```

Comment:
there wasn't any `text-gray-700` class to remove? Is that the default text color then?